Exec/inputs_compare_std2

```diff
@@ -1,33 +1,35 @@
+###
 n_cell = 100 20 2   
 max_grid_size_x = 100
 max_grid_size_y = 20
 max_grid_size_z = 2
 
 dt = 5.e-15  # MATLAB code uses 5.e-6 nanoseconds
-nsteps = 100000
+nsteps = 600000
 plot_int = 10000
 chk_int = -1
 restart = -1
-diag_type = 4
+diag_type = 2
+Hbias_sweep = 1
 
 prob_lo = 0. 0. 0.
 prob_hi = 500.e-9 100.e-9 10.e-9
 
 mu0 = 1.25663e-6
 
 Mx_parser(x,y,z) = "8.e5 * (x>0.)*(x<500.e-9)*(y>0.)*(y<100.e-9)*(z>0.)*(z<10.e-9)"
-My_parser(x,y,z) = "0."
-Mz_parser(x,y,z) = "0."
+My_parser(x,y,z) = "8.e5 * (x>0.)*(x<500.e-9)*(y>0.)*(y<100.e-9)*(z>0.)*(z<10.e-9)"
+Mz_parser(x,y,z) = "8.e5 * (x>0.)*(x<500.e-9)*(y>0.)*(y<100.e-9)*(z>0.)*(z<10.e-9)"
```

Comment:
Be careful - is Ms=8.e5 then you need the magnitude of M initially to also be 8.5e, i.e., sqrt(Mx^2 + My^2 + Mz^2) = Ms.  I can't find what direction the initial field should point (yet) so just point it purely in the x direction?